lib/tasks/migrate_labware_audits.rake

```diff
@@ -5,7 +5,7 @@ namespace :labware_events do
   desc 'Move the labware audits to the events warehouse'
   task create: :environment do |_t|
     # any audits before this date will have events created
-    end_date = DateTime.parse(ENV['END_DATE'])
+    end_date = DateTime.parse(ENV.fetch('END_DATE', nil))
```

Comment:
It may be clearer if you don't have a default value here:
```ruby
irb(main):002:0> DateTime.parse(ENV.fetch('END_DATE'))
Traceback (most recent call last):
        3: from (irb):1
        2: from (irb):2:in `rescue in irb_binding'
        1: from (irb):2:in `fetch'
KeyError (key not found: "END_DATE")
irb(main):003:0> DateTime.parse(ENV.fetch('END_DATE',nil))
Traceback (most recent call last):
        3: from (irb):2
        2: from (irb):3:in `rescue in irb_binding'
        1: from (irb):3:in `parse'
TypeError (no implicit conversion of nil into String)
```